<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1337</width>
    <height>914</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Arial Rounded MT Bold</family>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">*{
border:none;
padding: 0px;
margin:0px;
color:#fff
}

#centralwidget{
background-color: black;
}

#icon_only_widget, #full_menu_widget, #main_widget, #listWidget{
background-color:#1f1f1f;
border-radius:6px
}

#icon_only_widget QPushButton {
border:none;
padding:10px 30px;
border-radius:10px
}

#icon_only_widget QPushButton:hover {
border-radius:0px;
background-color: rgba(236, 236, 236, 0.1);
}

#full_menu_widget QPushButton {
border:none;
padding:10px 30px;
border-radius:10px;
color: gray
}

#full_menu_widget QPushButton:hover {
border-radius:0px;
background-color: rgba(236, 236, 236, 0.1);
}

#full_menu_widget QPushButton:checked {
color:white;
}

#search_input{
border:none;
padding: 15px 10px;
border-radius: 20px;
background-color: #2A2A2A;
color:white;
}

#search_input:hover {
border: 1px solid gray
}

#search_input:focus {
border: 2px solid white
}

#login_btn{
background-color:white;
padding: 10px 20px;
border-radius: 20px;
color:#121212;
}

#signup_btn{
padding: 10px;
color: #A2A2A2;
}

#signup_btn:hover{
color:white
}</string>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>10</number>
    </property>
    <property name="topMargin">
     <number>10</number>
    </property>
    <property name="rightMargin">
     <number>10</number>
    </property>
    <property name="bottomMargin">
     <number>10</number>
    </property>
    <property name="spacing">
     <number>10</number>
    </property>
    <item row="0" column="0">
     <widget class="QWidget" name="icon_only_widget" native="true">
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <layout class="QHBoxLayout" name="logo_layout">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item alignment="Qt::AlignVCenter">
          <widget class="QLabel" name="label">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>60</width>
             <height>60</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Arial Rounded MT Bold</family>
             <pointsize>5</pointsize>
            </font>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap>resources3.qrc/icons8-music-50.png</pixmap>
           </property>
           <property name="scaledContents">
            <bool>true</bool>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
           <property name="margin">
            <number>10</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <property name="spacing">
          <number>30</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>20</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QPushButton" name="home_btn">
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>resources3.qrc/House.svg</normaloff>resources3.qrc/House.svg</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>40</width>
             <height>40</height>
            </size>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="autoExclusive">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="library_btn">
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>resources3.qrc/Playlist.svg</normaloff>resources3.qrc/Playlist.svg</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>40</width>
             <height>40</height>
            </size>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="autoExclusive">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_4">
           <property name="text">
            <string>unknow</string>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="autoExclusive">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_5">
           <property name="text">
            <string>unknow</string>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="autoExclusive">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>522</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QWidget" name="full_menu_widget" native="true">
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <property name="spacing">
        <number>5</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="font">
          <font>
           <family>Arial Rounded MT Bold</family>
           <pointsize>30</pointsize>
          </font>
         </property>
         <property name="text">
          <string>MUSIC</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <property name="margin">
          <number>0</number>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <property name="spacing">
          <number>30</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>20</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QPushButton" name="home_btn_2">
           <property name="font">
            <font>
             <family>Arial Rounded MT Bold</family>
             <pointsize>15</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Home</string>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>resources3.qrc/House.svg</normaloff>resources3.qrc/House.svg</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>40</width>
             <height>40</height>
            </size>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="autoExclusive">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="library_btn_2">
           <property name="font">
            <font>
             <family>Arial Rounded MT Bold</family>
             <pointsize>15</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Your library</string>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>resources3.qrc/Playlist.svg</normaloff>resources3.qrc/Playlist.svg</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>40</width>
             <height>40</height>
            </size>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="autoExclusive">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_10">
           <property name="text">
            <string>unknow</string>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="autoExclusive">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_11">
           <property name="text">
            <string>unknow</string>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="autoExclusive">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>511</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QWidget" name="main_widget" native="true">
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QWidget" name="header_page" native="true">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>40</height>
          </size>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QPushButton" name="menu_btn">
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>resources3.qrc/icons8-menu-50.png</normaloff>resources3.qrc/icons8-menu-50.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>40</width>
              <height>40</height>
             </size>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Fixed</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>150</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <property name="rightMargin">
             <number>30</number>
            </property>
            <item>
             <widget class="QLineEdit" name="search_input">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>0</height>
               </size>
              </property>
              <property name="font">
               <font>
                <family>Arial Rounded MT Bold</family>
                <pointsize>10</pointsize>
               </font>
              </property>
              <property name="placeholderText">
               <string>What do you want to listen to?</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="search_btn">
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset>
                <normaloff>resources3.qrc/icons8-search-50.png</normaloff>resources3.qrc/icons8-search-50.png</iconset>
              </property>
              <property name="iconSize">
               <size>
                <width>40</width>
                <height>40</height>
               </size>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Fixed</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>100</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QPushButton" name="signup_btn">
              <property name="font">
               <font>
                <family>Arial Rounded MT Bold</family>
                <pointsize>10</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Sign up</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="login_btn">
              <property name="font">
               <font>
                <family>Arial Rounded MT Bold</family>
                <pointsize>10</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Log in</string>
              </property>
              <property name="iconSize">
               <size>
                <width>40</width>
                <height>40</height>
               </size>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QStackedWidget" name="stacked_widget">
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="home_widget">
          <layout class="QVBoxLayout" name="verticalLayout_7">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_6">
             <item>
              <widget class="QListWidget" name="listWidget">
               <property name="font">
                <font>
                 <family>Arial Rounded MT Bold</family>
                 <pointsize>14</pointsize>
                </font>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="text">
               <property name="font">
                <font>
                 <family>Arial Rounded MT Bold</family>
                 <pointsize>14</pointsize>
                </font>
               </property>
               <property name="text">
                <string>TextLabel</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="library_widget">
          <layout class="QVBoxLayout" name="verticalLayout_8">
           <item>
            <widget class="QLabel" name="label_2">
             <property name="font">
              <font>
               <family>Arial Rounded MT Bold</family>
               <pointsize>20</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Library page</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>menu_btn</sender>
   <signal>toggled(bool)</signal>
   <receiver>icon_only_widget</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>438</x>
     <y>47</y>
    </hint>
    <hint type="destinationlabel">
     <x>23</x>
     <y>800</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>menu_btn</sender>
   <signal>toggled(bool)</signal>
   <receiver>full_menu_widget</receiver>
   <slot>setHidden(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>438</x>
     <y>46</y>
    </hint>
    <hint type="destinationlabel">
     <x>339</x>
     <y>412</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>home_btn</sender>
   <signal>toggled(bool)</signal>
   <receiver>home_btn_2</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>59</x>
     <y>151</y>
    </hint>
    <hint type="destinationlabel">
     <x>325</x>
     <y>152</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>home_btn_2</sender>
   <signal>toggled(bool)</signal>
   <receiver>home_btn</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>244</x>
     <y>152</y>
    </hint>
    <hint type="destinationlabel">
     <x>71</x>
     <y>103</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>library_btn</sender>
   <signal>toggled(bool)</signal>
   <receiver>library_btn_2</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>70</x>
     <y>238</y>
    </hint>
    <hint type="destinationlabel">
     <x>182</x>
     <y>234</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>library_btn_2</sender>
   <signal>toggled(bool)</signal>
   <receiver>library_btn</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>247</x>
     <y>242</y>
    </hint>
    <hint type="destinationlabel">
     <x>87</x>
     <y>241</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
